src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -18,6 +18,10 @@ import styles from '../../../styles/styles';
 import * as CardUtils from '../../../libs/CardUtils';
 import Button from '../../../components/Button';
 import CardDetails from './WalletPage/CardDetails';
+// eslint-disable-next-line rulesdir/no-api-in-views
+import * as API from '../../../libs/API';
```

Comment:
This is here for a reason. If you want to call an API you need to create an "action" in the `Card.js` file